request/builder.js

```diff
@@ -66,8 +87,15 @@ class CertificateRegistrationRequestBuilder {
   /**
    * Builds the CertificateRegistrationRequest
    * @return {CertificateRegistrationRequest}
+   * @throws {Error}
    */
   async build() {
+    const validator = new Validator();
+    validator.validateInput(this.certHolderId, String);
+    validator.validateInput(this.certVersion, Number);
+    validator.validateInput(this.certPem, String);
+
```

Comment:
Please  remove a blank llne. (since doubled blank lines)

Reply:
@penpyt Please fix it the it's good to go.

Reply:
@feeblefakie 
I removed it.